src/KafkaFlow/Middlewares/Compressor/CompressorConsumerMiddleware.cs

```diff
@@ -1,22 +1,22 @@
-﻿namespace KafkaFlow.Compressor
+﻿namespace KafkaFlow
```

Comment:
The class namespaces should respect the folder hierarchy. The only exception is the configuration extension methods.

Reply:
Done